### ISSUE TITLE
Postman closes if incorrect passphrase is used for a key file

### ISSUE THREAD

**Issue by peter-janssen** [a85/POSTMan-Chrome-Extension#5765]
**Describe the bug**
Postman on debian closes if incorrect passphrase is used for a key file.

**To Reproduce**
1. Go to the 'Certificates' tab in 'Settings' 
2. Click on 'Add certificate' and fill in hostname, certificate path, key path and an _incorrect_ passphrase
3. Close 'Settings' and try to call an endpoint needing tls
4. The app chrashes and closes.

**Expected behavior**
Halting the operation and a warning or error stating the passphrase is incorrect.

**App information (please complete the following information):**
 - Native App
 - Postman Version: 6.7.1 - linux 4.9.0-8-amd64 / x64
 - OS: Debian GNU/Linux 9.6 (stretch)

**Additional context**
Note: SSL certificate verification is turned off

**Logging**
```
[3726][1547190431193][main][info]["Booting Postman 6.7.1, linux-4.9.0-8-amd64 on x64"]
[3726][1547190431197][main][info]["EventBus\\x7einitialize - Success"]
[3726][1547190431200][main][info]["UpdateHandler\\x7einit - Success"]
[3726][1547190431201][main][info]["RuntimeExecutionService\\x7einitialized: Success"]
[3726][1547190431211][main][info]["ProtocolHandler\\x7einit - Success with status: false]"]
[3726][1547190431318][main][info]["Bootstrap-models\\x7ebootstrap - Success"]
[3726][1547190435071][main][info]["Main\\x7eAppEvents - Received booted event for process shared"]
[3726][1547190439938][main][info]["UpdateHandler\\x7eapp-update-events - Received event",{"name":"checkForElectronVersionUpdated","namespace":"appUpdate"}]
[3726][1547190439941][main][info]["Main\\x7eAppEvents - Received booted event for process requester"]
[3726][1547190444216][main][error]["Main\\x7ehandleUncaughtError - Uncaught errors",{"name":"Error","message":"error:06065064:digital envelope routines:EVP_DecryptFinal_ex:bad decrypt","stack":"Error: error:06065064:digital envelope routines:EVP_DecryptFinal_ex:bad decrypt\n    at Object.createSecureContext (_tls_common.js:105:17)\n    at Object.exports.connect (_tls_wrap.js:1038:48)\n    at /opt/Postman/app/resources/app/node_modules/tunnel-agent/index.js:202:28\n    at ClientRequest.onConnect (/opt/Postman/app/resources/app/node_modules/tunnel-agent/index.js:163:7)\n    at Object.onceWrapper (events.js:320:30)\n    at emitThree (events.js:135:13)\n    at ClientRequest.emit (events.js:216:7)\n    at Socket.socketOnData (_http_client.js:486:11)\n    at emitOne (events.js:115:13)\n    at Socket.emit (events.js:210:7)\n    at addChunk (_stream_readable.js:252:12)\n    at readableAddChunk (_stream_readable.js:239:11)\n    at Socket.Readable.push (_stream_readable.js:197:10)\n    at TCP.onread (net.js:589:20)"}]
```

**Comment by numaanashraf** [comment]
@peter-janssen Thanks for reporting this. We are having some trouble replicating this at our end. Would it be possible to share more information on the client certificate? Specifically the cipher used to create the client key.

<img width="521" alt="screenshot 2019-02-01 at 2 38 04 pm" src="https://user-images.githubusercontent.com/3295568/52113556-a714fe00-262f-11e9-9da2-9bf1b4d02505.png">

**Comment by saswatds** [comment]
Fixed in Postman v7.2.0
